### DISCUSSION TITLE
Ports management (enable/disable)

### DISCUSSION THREAD

**Discussion by txitxo0** [vmakeev/huawei_mesh_router#73] [Ideas]
Hi there, just discovered this integration and working like a charm for my Honor Router 3! 

I was thinking if it is in the roadmap a fetaure to add  switchs for enabling and disabling ports forwarding. It would be great to be able to automate things with home assistant related to this things (i.e. if I start a specific  addon and I am outside home, open the port.

great work!

**Comment by vmakeev** [comment]
Hi, @txitxo0 ! 
Thank you for your kind words and for the great idea!

I have added the ability to control port forwarding switches to the [pre-release version of the component](https://github.com/vmakeev/huawei_mesh_router/releases/tag/v0.9.2-rc1). 
If it doesn't bother you, please install it and let me know if everything is working correctly.

**Reply by txitxo0** [reply]
Uau!! You are very kind and so fast!

I have just tested. And.....it is working like a charm!


I was trying to inspect the API and tried to develop before I saw you project. But my lack of knowledge in the auth side was determinant to can't prooced and go further.

Soany thanks in name of all the community. I guess this feature is as nice as the others you already developed. Thank you!
